src/main/kotlin/com/mjucow/eatda/common/vo/deseializer/Latitude.kt

```diff
@@ -0,0 +1,15 @@
+package com.mjucow.eatda.common.vo.deseializer
```

Comment:
vo로 바로 써도 될 것 같아서 패키지 이동해야될 것 같아요
```suggestion
package com.mjucow.eatda.common.vo
```

Reply:
넵